### PR TITLE
Yak shaving PR text

### LINKED ISSUE #1516: Lower the PR doc's text density
Currently https://docs.travis-ci.com/user/pull-requests (and possibly other intro docs) is quite dense. TOC and some reorganization would be helpful.

A related issue is a common confusion as to why two builds are triggered when a PR is opened. A link from the repo's Settings page to `/user/pull-requests` explaining why could be useful.

### PR DESCRIPTION
**Pull request by plaindocs**
Attempt to close  #1516 

Going to add Build PRs and Pushes info in #1685

### CODE REVIEW

**Comment by plaindocs** [comment]
Absolutely, thanks for the reminder.

So what we'er saying is that for the PR we show the PR build and the branch build. And for a heavily PR based work flow it might makes sense to turn the branch build off and see only one?

**Comment by plaindocs** [comment]
The two links in aaeda6e will fail untill #1685 is merged.

**Comment by gustamora** [comment]
@git_gustamora

File: user/pull-requests.md [hunk 1]
```diff
@@ -84,3 +66,14 @@ The command "eval git fetch origin +refs/pull/994/merge: " failed
 ```
 
 Restoring the branch of a merged pull request will not trigger a build, nor will pushing a new commit to a branch that has already been merged.
+
+## 'Double builds' on pull requests
+
+If you see two build status icons on your GitHub pull request, it means there is one build for the branch, and one build for the pull request itself (actually the build for the merge of the head branch with the base branch specified in the pull request).
+
+[Build pushed branches](/user/web-ui/#build-pushed-branches) and [Build pushed pull requests](/user/web-ui/#build-pushed-pull-requests) control this behaviour.
```
Comment (BanzaiMan) [inline user/pull-requests.md]: Nitpick: Are we standardizing on American or British spelling?

Comment (BanzaiMan) [reply user/pull-requests.md]: These links are dead. Are they meant to be added with this PR, or are they meant to point to something else? I looked around, but this wasn't clear to me.

Comment (plaindocs) [reply user/pull-requests.md]: https://github.com/travis-ci/docs-travis-ci-com/pull/1682#issuecomment-362588202 mentions these

Comment (plaindocs) [reply user/pull-requests.md]: In fact, @BanzaiMan if you wanna take a look at that PR as well... we can merge that one then this one.

Comment (plaindocs) [reply user/pull-requests.md]: On the spelling issue, that is a ball I've been avoiding. :-p Pretty sure we should be using American spelling, but I've not been putting huge effort into checking that.

There should be a linter for this, no?

**Pull request merged**